src/Files.App/Data/Commands/Manager/CommandCodes.cs

```diff
@@ -186,10 +186,13 @@ public enum CommandCodes
 		PreviousTab,
 		NextTab,
 		CloseSelectedTab,
+
+		// Shell Panes
 		OpenNewPane,
```

Comment:
Will this action be removed?

Reply:
No.

Reply:
I thought it's replaced by the vertical and horizontal actions.

Reply:
@mdtauk Since I haven't seen Open new pane and have seen Close active pane in your concepts, I thought there's still Open new pane. What should I do?

Reply:
The "Split pane" menu is replacing "Open new pane"

![image](https://github.com/files-community/Files/assets/39923744/6808fc99-9107-4e5c-a32c-acf7223b7680)

Reply:
Ah, then yes.